main.py

```diff
@@ -1,10 +1,36 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for player in data.keys():
```

Comment:
The loop iterates over data.keys(), which implies that the JSON structure is expected to have player names as keys. Ensure that this is the case in your 'players.json' file.